include/partitioning/parmetis_partitioner.h

```diff
@@ -117,7 +117,7 @@ class ParmetisPartitioner : public Partitioner
   /**
    * Maps active element ids into a contiguous range, as needed by ParMETIS.
    */
-  vectormap<dof_id_type, dof_id_type> _global_index_by_pid_map;
```

Comment:
Looks like `ParmetisPartitioner` and `MetisPartitioner` might be the only places where `vectormap` is currently used. If we eventually replace the one in `MetisPartitioner` as well, I think we should probably deprecate the whole class and remove it completely some day?

Reply:
Hmm... maybe.  IIRC Ben wrote it for a use where std::map would have been inefficient at a time when we weren't guaranteed to have unordered_map available, so maybe it's obsolete now?

I'd like to swap it in and out and test performance vs unordered_map on a few systems before we go to that trouble, though, I think.